pkg/plugin/aggregation/run.go

```diff
@@ -189,6 +217,11 @@ func Run(client kubernetes.Interface, plugins []plugin.Interface, cfg plugin.Agg
 	}
 }
 
+func canStartPlugin(a *Aggregator, p plugin.Interface) bool {
```

Comment:
Hi @johnSchnake,

Just curious, why isn't this a private function on `*Aggregator`?

Reply:
No reason; 6 on one hand half a dozen on the other IMO. Under the hood its all the same so it just came down to my random train of thought at the moment of writing it.